awswrangler/s3/_write_text.py

```diff
@@ -673,7 +673,6 @@ def to_csv(  # pylint: disable=too-many-arguments,too-many-locals,too-many-state
                 create_table_args["serde_parameters"] = serde_info.get("Parameters", None)
                 catalog._create_csv_table(**create_table_args)  # pylint: disable=protected-access
```

Comment:
That logging call is already made within that method [itself](https://github.com/aws/aws-sdk-pandas/blob/release-3.0.0/awswrangler/catalog/_create.py#L147) which is better

It's set to debug at the moment and I agree that it might be worth changing to info, but I don't want to do it in this PR